Polyhedron/demo/Polyhedron/MainWindow.h

```diff
@@ -354,6 +354,7 @@ protected Q_SLOTS:
   void updateMenus();
   void load_plugin(QString names, bool blacklisted);
   void recurseExpand(QModelIndex index);
+  void recomputeItems();
```

Comment:
`recomputeItems` is called only once. Just remove it and inline its body in the caller. That will avoid the recompilation of all plugins (only items will be recompiled).